ts/features/bonus/siciliaVola/screens/voucherGeneration/ko/SvSelectBeneficiaryCategoryKoScreen.tsx

```diff
@@ -1,35 +1,44 @@
 import * as React from "react";
 import { Dispatch } from "redux";
 import { connect } from "react-redux";
-import { View } from "native-base";
+import { SafeAreaView } from "react-native";
 import { InfoScreenComponent } from "../../../../../../components/infoScreen/InfoScreenComponent";
 import { renderInfoRasterImage } from "../../../../../../components/infoScreen/imageRendering";
 import image from "../../../../../../../img/servicesStatus/error-detail-icon.png";
 import I18n from "../../../../../../i18n";
-import { Body } from "../../../../../../components/core/typography/Body";
 import { GlobalState } from "../../../../../../store/reducers/types";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
+import { cancelButtonProps } from "../../../../bonusVacanze/components/buttons/ButtonConfigurations";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import { svGenerateVoucherCancel } from "../../../store/actions/voucherGeneration";
 
-const SvCheckIncomeKoScreen = () => (
-  <>
-    <View spacer={true} extralarge={true} />
-    <View spacer={true} extralarge={true} />
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const SvCheckIncomeKoScreen: React.FC<Props> = (props: Props) => (
```

Comment:
```suggestion
const SvCheckIncomeKoScreen: React.FC<Props> = (props: Props) => (
```
Could the screen name and its file match?
This is a little misleading

Reply:
Sure, related to the previous reporting.
Aligned in [9198089](https://github.com/pagopa/io-app/pull/3273/commits/9198089643bd48e55bca81c2e7a5cc2dd240c10e)